tests/api/test_addon_uploads.py

```diff
@@ -962,6 +962,44 @@ def test_extension_invalid_name(base_url, session_auth):
             ), f'Actual response message was {edit_addon.text}'
 
 
+@pytest.mark.parametrize(
+    'trademark_name',
+    [
+        'A Name with Firefox',
+        'A Name with Mozilla',
+        'A Name with Mozilla Firefox',
+        'Name has FireFox',
+        'Name has MOZILLA',
```

Comment:
Could we just do "Mozilla Firefox"? Do we know the exact filter for this?

Reply:
We want to cover as many edge cases as possible, this is why I'm using all these combinations. For example, we want to make sure that all or some uppercase letters do not impact the detection (we had issues in the past when upper case letters caused issues in some areas, so better safe than sorry). Also, we want to make sure that the trademark name is detected both in isolation and when combined with another trademark.